hosts

```diff
@@ -0,0 +1,14 @@
+[ansible]
```

Comment:
The group should be "control" and the node named "ansible" to be consistent with other parts.

Reply:
Also agreed here.